Fix CI failure by updating mavlink enums

Jaeyoung-Lim opened the pull request:
Upstream mavlink https://github.com/mavlink/mavlink/pull/1112 updated the frame unums, therefore the enums used in this repo didn't exist anymore.

CI was only breaking on macOS since this was the only one using upstream mavlink c library.

This commit updates the container with the updated enums

LorenzMeier commented on src/gazebo_mavlink_interface.cpp:
```diff
@@ -1229,7 +1229,7 @@ void GazeboMavlinkInterface::VisionCallback(OdomPtr& odom_message) {
     odom.time_usec = odom_message->time_usec();
 
     odom.frame_id = MAV_FRAME_LOCAL_NED;
-    odom.child_frame_id = MAV_FRAME_BODY_FRD;
```
The issue is here: The reason the frame got reserved is that it got stripped from MAVLink. So instead of changing the enum, the whole point is to change this for good.

Jaeyoung-Lim replied:
Do you mean by not using the enum at all?

TSC21 replied:
@Jaeyoung-Lim on the enum you can see this was deprecated and was replaced by another value.

TSC21 replied:
@Jaeyoung-Lim supposedly this was replaced by MAV_FRAME_BODY_OFFSET_NED -https://mavlink.io/en/messages/common.html#MAV_FRAME_RESERVED_12.

LorenzMeier reviewed (approved):
This is acceptable as an interim step. The full fix will be to ditch any non-standard frames and transform the data into one of the standard frames supported and/or moving these "internal" messages to FastRTPS.

LorenzMeier reviewed (changes requested):
The last commit broke it. It needs to be either FRD or RESERVED12 if not present.

Jaeyoung-Lim committed:
Update mavlink enums

mavlink was updated, therefore the enums used didn't exist anymore. However, the CI was only breaking on macOS since this was the only one using upstream mavlink c library.

This commit is a intermim solution to support both mavlink versions
